train/tr10-13B-ml/tr10-13B.slurm

```diff
@@ -81,7 +81,7 @@ GPT_ARGS=" \
     --micro-batch-size $MICRO_BATCH_SIZE \
     --rampup-batch-size 16 16 6_000_000 \
     --global-batch-size $GLOBAL_BATCH_SIZE \
-    --train-samples $((3000000000 / $SEQ_LEN + 1)) \
+    --train-samples $((300_000_000_000 / $SEQ_LEN + 1)) \
```

Comment:
except it doesn't work in bash:

```
bash: 300_000_000_000: value too great for base (error token is "300_000_000_000")
```

The easiest I found is to use Perl:

```
SEQ_LEN=2048
TRAIN_TOKENS=300_000_000_000
TRAIN_SAMPLES=$(perl -e "print $TRAIN_TOKENS / $SEQ_LEN")
[...]
--train-samples $TRAIN_SAMPLES
```

If you prefer python, so that everybody understands what exactly happens, then you have to add `int`:
```
python -c "print(int($TRAIN_TOKENS / $SEQ_LEN))"
```
either way works for me.

Reply:
I knew there was a reason I dropped that previous formatting ... thanks!

Reply:
actually you don't need to add `int` you should be able to run:
```
TRAIN_SAMPLES=$(python -c "print($TRAIN_TOKENS // $SEQ_LEN)")
```
`//` does euclidean division and returns the quotient.

Reply:
the new version looks great!